test/libsolidity/SolidityParser.cpp

```diff
@@ -934,6 +934,24 @@ BOOST_AUTO_TEST_CASE(library_simple)
 	BOOST_CHECK_NO_THROW(parseText(text));
 }
 
+BOOST_AUTO_TEST_CASE(multi_variable_declaration)
+{
+	char const* text = R"(
+		library Lib {
+			function f() {
+				var (a,b,c) = g();
+				var (d) = 2;
+				var (,e) = 3;
+				var (f,) = 4;
+				var (x,,) = g();
+				var (,y,) = g();
+			}
```

Comment:
what about
test for <... function f(); var () = f(); var (a, h) = f() ...> where f does not have return value.
or        <... function f() returns (uint, uint); var() = f(); var(, ,) = f(); var(a, b, c) = f(); ..>